quilt/tools/main.py

```diff
@@ -153,6 +155,8 @@ def check_hash(group, hashstr):
         return 0
     except command.CommandException as ex:
         print(ex, file=sys.stderr)
+        print()
+        parser.print_help()
```

Comment:
This isn't quite what we want because it prints the top-level help for the quilt command. This will print the specific help for the command the user ran (and triggered the exception):

cmd = kwargs['cmd']
print(subparsers.choices[cmd].format_help())